pages/partnerships/partnerships.html

```diff
@@ -0,0 +1,422 @@
+        </p>
+        <p>
+          We’ll get back to you right away!
+        </p>
+      </md-content>
+    </md-card-content>
+    <div flex-gt-sm="10"></div>
+    <md-card class="oppia-foundation-white-background oppia-foundation-black-text md-no-momentum oppia-foundation-vertical-padding" flex-gt-sm="45">
+      <form name="userContactsForm" class="oppia-foundation-white-background">
+        <md-input-container class="md-block">
+          <label style="text-transform: uppercase;">Name</label>
+          <md-icon md-svg-src="/static/icons/baseline-person-24px.svg"></md-icon>
+          <input placeholder="" required md-no-asterisk name="name" ng-model="userContacts.name">
+          <div ng-messages="userContactsForm.name.$error">
+            <div ng-message="required">Please enter your full name.</div>
+          </div>
+        </md-input-container>
+        <md-input-container class="md-block">
+          <label style="text-transform: uppercase;">Email</label>
+          <md-icon md-svg-src="/static/icons/baseline-email-24px.svg"></md-icon>
+          <input placeholder="" required md-no-asterisk name="email" md-maxlength="100" ng-model="userContacts.email" ng-pattern="/^.+@.+\..+$/" minlength="10" maxlength="100">
+          <div ng-messages="userContactsForm.email.$error" multiple>
+            <div ng-message="required">Email address is required.</div>
+            <div ng-message="pattern">Please enter a valid email address</div>
+            <div ng-message="minlength">Email must be longer than 10 characters</div>
+            <div ng-message="maxlength">Email must be between 10 and 100 characters</div>
+          </div>
+        </md-input-container>
+        <md-input-container class="md-block">
+          <label style="text-transform: uppercase;">Organization</label>
+          <md-icon md-svg-src="/static/icons/baseline-business-24px.svg"></md-icon>
+          <input placeholder="" md-no-asterisk name="organization" ng-model="userContacts.organization">
+        </md-input-container>
+        <md-input-container class="md-block">
+          <label><span style="text-transform: uppercase">Message</span> (Optional)</label>
+          <md-icon md-svg-src="/static/icons/baseline-comment-24px.svg"></md-icon>
+          <textarea ng-model="userContacts.comment" name="comment" md-maxlength="150" rows="5" md-select-on-focus></textarea>
+          <div ng-messages="userContactsForm.comment.$error">
+            <div ng-message="md-maxlength">We appreciate your opinion but please do not go over the word limit! Consider email us directly.</div>
+          </div>
+        </md-input-container>
+        <md-button class="oppia-foundation-button-horizontal-padding oppia-foundation-green-button" type="submit">Submit</md-button>
+      </form>
+    </md-card>
+  </md-card>
+</section>
+
+<style>
+  blockquote {
```

Comment:
If this is generic it should go in the main CSS stylesheet.

Reply:
Upon closer inspection, I don't think I want to override blockquote in general so I turned this into a specific css class.